lib/index.d.ts

```diff
@@ -1,31 +1,16 @@
-// Type definitions for iconv-lite
-// Project: https://github.com/ashtuchkin/iconv-lite
-// Definitions by: Martin Poelstra <https://github.com/poelstra>
-// Definitions: https://github.com/borisyankov/DefinitelyTyped
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
 
-import stream = require("stream");
+declare module 'iconv-lite' {
```

Comment:
Not sure tbh. I'm no authority on type definitions. The main problem comes from writing definitions that Angular's AoT compiler understands.